### PR TITLE
Watch multiple kinds at once

### PR DESCRIPTION
**Pull request by jcantrill**
This PR:
* Allows you to create a single watch in a namespace on multiple kinds

### CODE REVIEW

**Comment by jcantrill** [comment]
[test]

**Comment by jcantrill** [comment]
@fbricon @adietish Thoughts?  Added to simplify watch in JBT

File: src/main/java/com/openshift/internal/restclient/WatchClient.java [hunk 1]
```diff
@@ -135,8 +147,12 @@ private WebSocketClient newWebSocketClient() {
 		return client;
 	}
 	
-	private String getResourceVersion(String kind, String namespace) throws Exception{
+	private String getResourceVersion(String kind, String namespace, WatchEndpoint endpoint) throws Exception{
 		IList list = client.get(kind, namespace);
+		Collection<IResource> items = list.getItems();
+		List<IResource> resources = new ArrayList<>(items.size());
```
Comment (fbricon) [inline src/main/java/com/openshift/internal/restclient/WatchClient.java]: new ArrayList<>(items);

File: src/main/java/com/openshift/restclient/IClient.java [hunk 2]
```diff
@@ -30,6 +30,8 @@
 public interface IClient extends ICapable, IAuthorizationClient{
 	
 	IWatcher watch(String kind, String namespace, IOpenShiftWatchListener listener);
+
+	IWatcher watch(Collection<String> kinds, String namespace, IOpenShiftWatchListener listener);
```
Comment (fbricon) [inline src/main/java/com/openshift/restclient/IClient.java]: I usually like using something like  `IWatcher watch(String namespace, IOpenShiftWatchListener listener, String...kinds)` so you treat single/multiple params the same

**Comment by jcantrill** [comment]
[merge]

**Pull request merged**